_wiki/sfmb_betatest_community_rules.md

```diff
@@ -42,6 +42,7 @@ Please respect people in server for effective development & conversation.
 - Respect channel descriptions
 
 ## Recommended things
+- If you use Hamachi or VPN, make sure to close it before you open the game
```

Comment:
This is not problem after v7.0
before 7.0: Game uses MAC address & IP address for verify people, but Hamachi changes MAC & IP.
after 7.0: Game checks MAC & IP but it's for reference because Discord ID is trustable.